src/biogeophys/CanopyFluxesMod.F90

```diff
@@ -1548,6 +1557,26 @@ subroutine CanopyFluxes(bounds,  num_exposedvegp, filter_exposedvegp,
          call PhotosynthesisTotal(fn, filterp, &
               atm2lnd_inst, canopystate_inst, photosyns_inst)
          
+         ! Calculate water use efficiency
+         !   does not support multi-layer canopy
+         if (nlevcan == 1) then 
+            do f = 1, fn
+               p = filterp(f)
+               c = patch%column(p)
+               g = patch%gridcell(p)
+               
+               if ( is_near_local_noon( grc%londeg(g), deltasec=3600 ) .and. fpsn(p)>0._r8 )then
```

Comment:
You could use isecsphr from clm_varcon here in place of the hardcoded 3600. But, that's not too big of a deal either way.